docs/configuration/http_conn_man/rds.rst

```diff
@@ -28,7 +28,7 @@ route_config_name
   *(required, string)* The name of the route configuration. This name will be passed to the
   :ref:`RDS HTTP API <config_http_conn_man_rds_api>`. This allows an Envoy configuration with
   multiple HTTP listeners (and associated HTTP connection manager filters) to use different route
-  configurations.
+  configurations. The route_config_name must **not** contain ``:``.
```

Comment:
Because if for example someone specifies route_config_name `foo` and cluster `bar` and someone else does route_config_name `foob` and cluster `ar` the concatenation would make the two providers collide when it shouldn't.